### DISCUSSION TITLE
keep the allsky map from resetting to all world view..

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#63] [General]
So when making changes to our webserver settings and then going to the allsky map (https://www.thomasjacquin.com/allsky-map/), its tiring to have to zoom scroll back to the locating being viewed. Is there a way to refresh without setting the zoom to "all planet'?

**Comment by Alex-developer** [comment]
@EricClaeys FYI

**Reply by EricClaeys** [reply]
@thomasjacquin, Thomas, any thoughts?

**Reply by thomasjacquin** [reply]
The map is initialized with this block of code.
```
const myLatLng = { lat: 30, lng: 0 };
const map = new google.maps.Map(document.getElementById("map"), {
    zoom: 2,
    center: myLatLng,
});
```
I could save `zoom` and `center` in local storage and restore them on the next page reload.

**Reply by EricClaeys** [reply]
@thomasjacquin, if that meets Steve's needs, I'm all for it.

p.s., nice to have you back...

**Comment by thomasjacquin** [comment]
This is what I propose https://www.thomasjacquin.com/allsky-map/index-dev.php
Let me know if this fits the requirements and I will deploy them to the map.

**Comment by thomasjacquin** [comment]
@EricClaeys I added a few more options to the map that users may find useful as well. Can you test the link above and let me know if you notice any bugs?

**Comment by ssagerian** [comment]
It works great. I do like the new features, adding the night/twilight zones
is fun to see what cameras in different zones are showing.
The dark mode is very easy on the eyes.
I dont think I ever noticed if there was that field(buttons) down in the
left bottom corner before now.

**Comment by EricClaeys** [comment]
I also like the features, which work well.
On my phone all the overlays (upper left, lower left, etc.) are small and hard to read.  This has always been the case, but now with the buttons it's a pain since It was almost impossible to press the correct button.

Eric
________________________________

**Comment by thomasjacquin** [comment]
The mobile version should be working better now. I deployed the changes.

**Comment by EricClaeys** [comment]
Works great @thomasjacquin. Thanks for the quick work.

@ssagerian, Steve, how do you like it?

**Comment by ssagerian** [comment]
Eric, works great! Easy to navigate, actually fun to explore other peoples
sites, Im looking forward to making it part of my presentation in February.
